VK_V1.0/VK_V1.0/PhotoScene/PhotoCollectionViewController.swift

```diff
@@ -41,6 +41,15 @@ final class PhotoCollectionViewController: UICollectionViewController {
         return cell
     }
 
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard segue.identifier == "swipeSceneSegue",
```

Comment:
```suggestion
        guard segue.identifier == "swipeSceneSegue",
```
"swipeSceneSegue", в константы